monkey/infection_monkey/utils/load_plugins.py

```diff
@@ -0,0 +1,41 @@
+import importlib
+import inspect
+import logging
+from os.path import dirname, basename, isfile, join
+import glob
+
+LOG = logging.getLogger(__name__)
+
+
+def _get_candidate_files(base_package_file):
+    """
+    Gets all files under current directory(/actions)
+    :return: list of all files without .py ending
+    """
+    files = glob.glob(join(dirname(base_package_file), "*.py"))
+    return [basename(f)[:-3] for f in files if isfile(f) and not f.endswith('__init__.py')]
+
+
+def get_instances(base_package_name, base_package_file, parent_class):
+    """
+    Returns the parent_class type objects from base_package_spec according to configuration
```

Comment:
This documentation seems incorrect. It's not promised that the should_run will be based on configuration. Seems like an LoD break.